scholly/base/package.json

```diff
@@ -11,6 +11,7 @@
     "react-bootstrap": "^1.6.0",
     "react-dom": "^17.0.2",
     "react-helmet": "^6.1.0",
+    "react-icon": "^1.0.0",
```

Comment:
not important but is it possible to find suitable icons that you need from `react-icons` library instead of introducing new `react-icon`

Reply:
Thanks for catching that, I think I imported the wrong library, will remove it